qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/ImportExportContactsUITest.java

```diff
@@ -233,16 +222,16 @@ public void testExportManyContact() throws JSONException {
         assertNotNull(file1);
         goToContacts();
         openActionBarOverflowOrOptionsMenu(InstrumentationRegistry.getTargetContext());
-
         Spoon.screenshot(mActivity, "exportAll");
-		pressBack();
 		Intent data = new Intent();
 		data.setData(Uri.fromFile(file1));
-		ContactFragment contactFragment = (ContactFragment) mActivity.getFragmentManager().findFragmentById(R.id.fragment_container);
+		ContactFragment contactFragment =
```

Comment:
Indentation looks weird, removing tabs and breaking after a dot may increase beauty here ^^